src/govuk/components/notification-banner/template.test.js

```diff
@@ -0,0 +1,327 @@
+
+      expect($content.html().trim()).toEqual('You have 9 days to send a response.')
+    })
+  })
+
+  describe('custom options', () => {
+    it('renders custom title', () => {
+      const $ = render('notification-banner', examples['custom title'])
+      const $title = $('.govuk-notification-banner__title')
+
+      expect($title.html().trim()).toEqual('Important information')
+    })
+
+    it('renders custom content', () => {
+      const $ = render('notification-banner', examples['custom text'])
+      const $content = $('.govuk-notification-banner__content')
+
+      expect($content.html().trim()).toEqual('This publication was withdrawn on 7 March 2014.')
+    })
+
+    it('renders custom heading level', () => {
+      const $ = render('notification-banner', examples['custom title heading level'])
+      const $title = $('.govuk-notification-banner__title')
+
+      expect($title.get(0).tagName).toEqual('h3')
+    })
+
+    it('renders custom role', () => {
+      const $ = render('notification-banner', examples['custom role'])
+      const $component = $('.govuk-notification-banner')
+
+      expect($component.attr('role')).toEqual('banner')
+    })
+
+    it('renders aria-labelledby attribute matching the title id when role overridden to region', () => {
+      const $ = render('notification-banner', examples['role overridden to region'])
+      const ariaAttr = $('.govuk-notification-banner').attr('aria-labelledby')
+      const titleId = $('.govuk-notification-banner__title').attr('id')
+
+      expect(ariaAttr).toEqual(titleId)
+    })
+
+    it('renders custom tabindex', () => {
+      const $ = render('notification-banner', examples['custom tabindex'])
+      const $component = $('.govuk-notification-banner')
+
+      expect($component.attr('tabindex')).toEqual('0')
+    })
+
+    it('renders custom title id', () => {
+      const $ = render('notification-banner', examples['custom title id'])
+      const $title = $('.govuk-notification-banner__title')
+
+      expect($title.attr('id')).toEqual('my-id')
+    })
+
+    it('has an aria-labelledby attribute matching the title id', () => {
+      const $ = render('notification-banner', examples['custom title id'])
+      const ariaAttr = $('.govuk-notification-banner').attr('aria-labelledby')
+
+      expect(ariaAttr).toEqual('my-id')
+    })
+
+    it('renders data-module attribute to focus component on page load', () => {
+      const $ = render('notification-banner', examples['initialFocus as true'])
+
+      const $component = $('.govuk-notification-banner')
+      expect($component.attr('data-module')).toEqual('govuk-initial-focus')
+    })
+
+    it('removes tabindex attribute so component is not focusable', () => {
+      const $ = render('notification-banner', examples['tabindex as false and type as success'])
+
+      const $component = $('.govuk-notification-banner')
+      expect($component.attr('tabindex')).toBeFalsy()
```

Comment:
Does checking if this is undefined work? This would pass if tabindex="0", for example.

```suggestion
      expect($component.attr('tabindex')).toBeUndefined()
```

Reply:
Could you double check this one please? 🙏 I had the test failing with `.toBeFalsy()` if I set `tabindex=0` or `tabindex="0"`.

Reply:
@hannalaakso I get the same, the test fails if tabindex is set to 0, so I think it's ok 👍